packages/node/src/lock/memory-lock-service.ts

```diff
@@ -0,0 +1,47 @@
+import { Lock } from "./lock";
+import { ILockInterface } from "./types";
+
+export class MemoryLockService implements ILockInterface {
```

Comment:
REMINDER: This is a _local_ implementation. It is not a central lock. It's just a dummy local lock.